.vitepress/theme/ChatGPTButton.vue

```diff
@@ -0,0 +1,354 @@
+          fill="currentColor"
+        >
+          <path
+            d="M6.5 3L11 7.5L6.5 12"
+            stroke="currentColor"
+            stroke-width="1.5"
+            fill="none"
+            stroke-linecap="round"
+            stroke-linejoin="round"
+          />
+        </svg>
+      </a>
+
+      <a
+        class="dropdown-item"
+        :href="claudeLink"
+        target="_blank"
+        rel="noopener"
+        @click="closeDropdown"
+      >
+        <div class="item-icon-wrapper">
+          <img src="/assets/anthropic.svg" alt="Anthropic" class="item-icon" />
+        </div>
+        <div class="item-content">
+          <div class="item-title">Open in Claude</div>
+          <div class="item-description">Ask questions about this page</div>
+        </div>
+        <svg
+          class="item-arrow"
+          width="16"
+          height="16"
+          viewBox="0 0 16 16"
+          fill="currentColor"
+        >
+          <path
+            d="M6.5 3L11 7.5L6.5 12"
+            stroke="currentColor"
+            stroke-width="1.5"
+            fill="none"
+            stroke-linecap="round"
+            stroke-linejoin="round"
+          />
+        </svg>
+      </a>
+
+      <button
+        class="dropdown-item"
+        @click="copyPageContent"
+        :disabled="isCopying"
+      >
+        <div class="item-icon-wrapper">
+          <svg
+            class="item-icon"
+            width="20"
+            height="20"
+            viewBox="0 0 24 24"
+            fill="currentColor"
+          >
+            <path
+              d="M16 1H4c-1.1 0-2 .9-2 2v14h2V3h12V1zm3 4H8c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h11c1.1 0 2-.9 2-2V7c0-1.1-.9-2-2-2zm0 16H8V7h11v14z"
+            />
+          </svg>
+        </div>
+        <div class="item-content">
+          <div class="item-title">
+            {{ isCopying ? 'Copied!' : 'Copy page' }}
+          </div>
+          <div class="item-description">Copy page as Markdown for LLMs</div>
+        </div>
+      </button>
+    </div>
+  </div>
+</template>
+
+<script setup>
+import { ref, computed, onMounted, onUnmounted } from 'vue';
+import { useRoute } from 'vitepress';
+import TurndownService from 'turndown';
+
+const route = useRoute();
+const dropdownRef = ref(null);
+const isDropdownOpen = ref(false);
+const isCopying = ref(false);
+
+// Initialize handleClickOutside ref
+const handleClickOutsideRef = ref(null);
+
+const chatGPTLink = computed(() => {
+  const fullUrl = `https://tools.docs.iex.ec${route.path}`;
+  const prompt = `Please research and analyze this page: ${fullUrl} so I can ask you questions about it. Once you have read it, prompt me with any questions I have. Do not post content from the page in your response. Any of my follow up questions must reference the site I gave you.`;
+  return `https://chatgpt.com/?hints=search&q=${encodeURIComponent(prompt)}`;
+});
+
+const claudeLink = computed(() => {
+  const fullUrl = `https://tools.docs.iex.ec${route.path}`;
```

Comment:
```suggestion
  const fullUrl = `https://protocol.docs.iex.ec${route.path}`;
```
same here, It might be interesting to retrieve its URL dynamically.